src/util/sdl_input_source.cpp

```diff
@@ -337,6 +340,61 @@ u32 SDLInputSource::ParseRGBForPlayerId(std::string_view str, u32 player_id)
   return color;
 }
 
+bool SDLInputSource::IsPS5Controller(SDL_Gamepad* gp)
+{
+  const char* supported_controllers[] = {"DualSense Wireless Controller", "DualSense Edge Wireless Controller"};
```

Comment:
I feel like there has to be a better way to identify this. Seems error-prone and/or not future proof - I know my PS5 controller has appeared under different device name depending on SDL hints.